src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/CSharp.xaml

```diff
@@ -50,6 +50,9 @@
     </StringProperty.Metadata>
   </StringProperty>
   <StringProperty Name="Link" Visible="false">
+    <StringProperty.DataSource>
```

Comment:
This is for Compile items. You need to update rules for Content, None, EmbeddedResource and AdditionalFiles as well.